soundfonts/src/sfz/mod.rs

```diff
@@ -54,11 +54,14 @@ pub struct RegionParamsBuilder {
     lovel: u8,
     hivel: u8,
     key: Option<u8>,
+    lokey: Option<u8>,
+    hikey: Option<u8>,
     pitch_keycenter: Option<u8>,
     pan: i8,
     sample: Option<String>,
     default_path: Option<String>,
     loop_mode: SfzLoopMode,
+    cutoff: f32,
```

Comment:
If we're making cutoff optional in the synth (e.g. different code is used for voices with and without cutoff), it might be a good idea to make this `Option<f32>`

Reply:
That would require if statements on each voice to check if there is cutoff.
Wouldn't that be slower than just applying the cutoff every time (because now we are not only applying cutoff but also running an if statement)?